.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,8 @@
     - [ ] Added comprehensive tests for new/modified functionality
     - [ ] Updated `CHANGELOG.md` for product changes (`features, bug fixes, UI/UX, updated dependencies`)
     - [ ] Spelling check passes: `.\eng\common\spelling\Invoke-Cspell.ps1`
-- [ ] For MCP tool changes, updated:
+- [ ] If adding an MCP tool, updated the docstring and/or documentation
```

Comment:
these should be indented 4 spaces under the ` - [ ] For MCP tool changes, updated:"